### PR TITLE
issue + PR templates

### PR DESCRIPTION
**Pull request by DalderupMaurice**
Added issue, PR templates and added contributions file + updated Readme

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Welcome to the nOS community! We're always looking for more contributors and are happy to have you. 
+
+Therefor a big thank you for being here! We really appreciate you taking the time for reading our guidelines! 
+
+# How to contribute
+## Reporting Bugs
+Well-written bug reports with consistently reproducible steps are invaluable to the development of NEO•ONE. Bugs are tracked as GitHub issues. Before creating an issue, please perform a search to see if the problem has already been reported. After you've determined a bug does not already exist, create an issue and fill in the template.
```
Comment (mhuggins) [inline .github/CONTRIBUTING.md]: `NEO•ONE` => `nOS`? 😏

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  **Scope** is the package folder that you are focusing on, `api`, `hocs`, `account`, etc.
+
+  Describe your changes within a short sentence. If there are breaking changes, Do start with `BREAKING CHANGE`.
+
+### And you are done!
+Once you've had your first pull request approved and merged, find existing issues marked with `help wanted` labels to continue contributing or suggest an enhancement. Respond on the issue thread expressing interest in working on it. This helps other people know that the issue is active, and hopefully prevents duplicated efforts.
+
+
+## License
+By contributing to nOS, you agree that your contributions will be licensed under its MIT license.
```
Comment (mhuggins) [inline .github/CONTRIBUTING.md]: We should probably add a license to the root folder and `package.json`.  Does MIT make sense for us?  I never know enough about these things. 😅

Comment (jeroenptrs) [reply .github/CONTRIBUTING.md]: I think @deanpress stated a few times that our repos would be MIT licensed, so I guess it's the best choice yeah. Let me update this repo 😄

**Comment by jeroenptrs** [comment]
@mhuggins @DalderupMaurice added MIT License and also referenced it to package.json. Also added Code of Conduct and fixed the Contributing 'typo'.

**Comment by DalderupMaurice** [comment]
I cannot approve my own PR 😂 
So it's up to @mhuggins to approve/merge :)

**Pull request merged**